packages/@sanity/default-layout/src/sideMenu/SideMenu.css

```diff
@@ -7,8 +7,10 @@
 }
 
 .root {
-  position: fixed;
-  z-index: var(--zindex-drawer);
+  /* Override Sanity UI */
```

Comment:
Same thing with maybe a mention of why and when we can remove it. Maybe it's not a temporary thing at all? 😊